src/components/RichTextEditor/RichTextEditor.stories.tsx

```diff
@@ -17,19 +19,46 @@ export default {
     },
 } as Meta;
 
+const value = JSON.stringify([
+    { type: BlockStyleTypes.Paragraph, children: [{ text: "bold", bold: true }] },
+    { type: BlockStyleTypes.Paragraph, children: [{ text: "italic", italic: true }] },
+    { type: BlockStyleTypes.Paragraph, children: [{ text: "underline", underline: true }] },
+    { type: BlockStyleTypes.Paragraph, children: [{ text: "strikethrough", strikethrough: true }] },
+    { type: BlockStyleTypes.Code, children: [{ text: "code" }] },
+    {
+        type: BlockStyleTypes.UnorderedList,
+        children: [createListItemNode("red"), createListItemNode("blue"), createListItemNode("yellow")],
+    },
+    {
+        type: BlockStyleTypes.OrderedList,
+        children: [
+            createListItemNode("Mix flour, baking powder, sugar, and salt."),
+            createListItemNode("In another bowl, mix eggs, milk, and oil."),
+            createListItemNode("Stir both mixtures together."),
+            createListItemNode("Fill muffin tray 3/4 full."),
+            createListItemNode("Bake for 20 minutes."),
+        ],
+    },
+    {
+        type: BlockStyleTypes.Paragraph,
+        children: [createLinkNode("https://git-scm.com/downloads", "Link")],
```

Comment:
Maybe only include the link in the WithReadonlyState story until the link chooser is implemented.
Otherwise it's confusing that it can't be edited.

Reply:
👍  good point

Reply:
4224818a091d64f8c514c1726bdfcd152674fc58 & a08419bae0380168e10210cdecb68053de5204d9